### PR TITLE
CASMPET-5786 1.3 : DOCS: Update postgres kubectl exec/logs calls to include postgres container

### PR DESCRIPTION
**Pull request by kimjensen-hpe**
# Description

Update docs to include postgres container as needed. 
Reviewed all the docs on the main branch and found a few places that also needed updates.
This is a follow on from CASMTRIAGE-3727

# Checklist Before Merging

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I don't have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by rustydb — APPROVED** [review]

File: operations/kubernetes/View_Postgres_Information_for_System_Databases.md [hunk 1]
```diff
@@ -41,30 +41,30 @@ This procedure requires administrative privileges.
     run: /etc/service/pgqd: (pid 25) 487273s
     ```
 
-2.  Log in as the postgres user.
+2. Log in as the `postgres` user.
```
Comment (mharding-hpe) [inline operations/kubernetes/View_Postgres_Information_for_System_Databases.md:44]:
```suggestion
1. Log in as the `postgres` user.
```

File: operations/kubernetes/View_Postgres_Information_for_System_Databases.md [hunk 2]
```diff
@@ -41,30 +41,30 @@ This procedure requires administrative privileges.
...
-    ```
+    ```text
     psql (12.2 (Ubuntu 12.2-1.pgdg18.04+1), server 11.7 (Ubuntu 11.7-1.pgdg18.04+1))
     Type "help" for help.
 
     postgres=#
     ```
 
-3.  List the existing databases.
+3. List the existing databases.
```
Comment (mharding-hpe) [inline operations/kubernetes/View_Postgres_Information_for_System_Databases.md:59]:
```suggestion
1. List the existing databases.
```

File: operations/kubernetes/View_Postgres_Information_for_System_Databases.md [hunk 3]
```diff
@@ -88,21 +88,21 @@ This procedure requires administrative privileges.
...
 
-    ```
+    ```text
     psql (12.2 (Ubuntu 12.2-1.pgdg18.04+1), server 11.7 (Ubuntu 11.7-1.pgdg18.04+1))
     You are now connected to database "hmsds" as user "postgres".
     hmsds=#
     ```
 
-5.  List the data types that are in the database being viewed.
+5. List the data types that are in the database being viewed.
```
Comment (mharding-hpe) [inline operations/kubernetes/View_Postgres_Information_for_System_Databases.md:97]:
```suggestion
1. List the data types that are in the database being viewed.
```

**Review by mharding-hpe — CHANGES_REQUESTED** [review]
Doing some linting

File: operations/kubernetes/View_Postgres_Information_for_System_Databases.md [hunk 4]
```diff
@@ -41,30 +41,30 @@ This procedure requires administrative privileges.
     run: /etc/service/pgqd: (pid 25) 487273s
     ```
 
-2.  Log in as the postgres user.
+1. (`cray-smd-postgres#`) Log in as the `postgres` user.
```
Comment (mharding-hpe) [inline operations/kubernetes/View_Postgres_Information_for_System_Databases.md:44]:
```suggestion
1. (`postgres-container#`) Log in as the `postgres` user.
```

**Review by rustydb — APPROVED** [review]

**Review by nrockershousen — APPROVED** [review]

**Pull request merged**